ops/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,3 @@
+---
+remote_user: "ubuntu"
```

Comment:
This is what ownCloud does:
```
root@mysql:/etc/apache2/conf-enabled# cat owncloud.conf
Alias /owncloud "/var/www/owncloud/"
<Directory "/var/www/owncloud">
  Options +FollowSymLinks
  AllowOverride All

  <IfModule mod_dav.c>
        Dav off
  </IfModule>

  SetEnv HOME /var/www/owncloud
  SetEnv HTTP_HOME /var/www/owncloud
</Directory>

<Directory "/var/www/owncloud/data/">
  # just in case if .htaccess gets disabled
    Require all denied
</Directory>
```